pysindy/feature_library/base.py

```diff
@@ -351,6 +339,9 @@ def get_feature_names(self, input_features=None):
             feature_names += lib_feat_names
         return feature_names
 
+    def calc_trajectory(self, diff_method, x, t):
+        return self.libraries_[0].calc_trajectory(diff_method, x, t)
+
```

Comment:
Why did the old method from the superclass not work?

Reply:
If some `NewFeatureLibrary` class overwrites `calc_trajectory` to include some special processing necessary for a SINDy fit, then we'd want to allow `ConcatLibrary`'s and `TensoredLibrary`'s composed of `NewFeatureLibrary`'s to utilize the `calc_trajectory` of the first library. This is done, for example, in the new `ParameterizedLibrary`, where the time derivatives are rescaled if the library is weak. If we tried to concatenate two weak `ParameterLibrary`'s, we want that rescaling to still take place in the `ConcatLibrary`. I know that's a little complicated, but I hope it makes sense...

Reply:
Makes perfect sense!